src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> convertedTransaction = new ArrayList<>();
+        for (int i = 1; i < data.size(); i++) { // Skip header
+            String[] parts = data.get(i).split(",");
+            if (parts.length != 3) {
```

Comment:
Consider replacing the magic number '3' with a named constant to improve code readability and maintainability.